cdc/changefeed.go

```diff
@@ -523,10 +532,20 @@ func (c *changeFeed) handleMoveTableJobs(ctx context.Context, captures map[model
 			job.Status = model.MoveTableStatusDeleted
 			log.Info("handle the move job, remove table from the source capture", zap.Reflect("job", job))
 		case model.MoveTableStatusDeleted:
+			// Do NOT dispatch tables before checkpoint ts has been flushed to Etcd.
+			if c.appliedCheckpointTs != c.status.CheckpointTs {
```

Comment:
Since the owner has checked `appliedCheckpointTs` against `c.status.CheckpointTs`, is it possible that `c.appliedCheckpointTs != c.status.CheckpointTs` happens

Reply:
It has happened during testing. Which line of code are you referring to by "owner has checked `appliedCheckpointTs` against `c.status.CheckpointTs`"?